matchzoo/data_pack/data_pack.py

```diff
@@ -294,6 +294,35 @@ def drop_label(self):
         """
         self._relation = self._relation.drop(columns='label')
 
+    @_optional_inplace
+    def drop_invalid(self):
+        """
+        Remove rows from the data pack where the content is invalid.
+
```

Comment:
You may want to explain the meaning of `invalid` here (it's length-based instead of content-based).

Reply:
ok～